protocol/go.mod

```diff
@@ -1,19 +1,17 @@
 module github.com/dydxprotocol/v4-chain/protocol
 
-go 1.22
-
-toolchain go1.22.0
```

Comment:
Why does this get removed?

Reply:
I set slinky version to v1.0.8 and ran go mod tidy and it made this change